src/tools/debugger/stepping/Stepping.java

```diff
@@ -0,0 +1,42 @@
+package tools.debugger.stepping;
+
+import tools.SourceCoordinate.FullSourceCoordinate;
+/**
+ *
+ * Manage the actor message stepping.
+ *
+ */
+public class Stepping {
+
+  public enum SteppingType{
+    STEP_INTO,
+    STEP_OVER,
+    STEP_RETURN
+  }
+
+  private StepActorOperation stepActor;
+
+  public Stepping(final StepActorOperation stepActor) {
+    this.stepActor = stepActor;
+  }
+  /**
+   * Return the stepping operation that has been made for a specific source section,
```

Comment:
I tried to explain in the comment why the comparison is not made with the sourceSection object, but only with one of its attributes, the start line. I probably need to think more about the combination of both stepping sequential and message-oriented, but as I see it now I think I need this comparison in order to execute message stepping operations when the previous step was sequential. For example, if you did a step-over and after you want to do a step-into-message operation, the sourceSection of the SuspendedEvent does not match with the sourceSection of the <-: